mat64/dense.go

```diff
@@ -379,7 +379,9 @@ func (m *Dense) Clone(a Matrix) {
 
 // Copy makes a copy of elements of a into the receiver. It is similar to the
 // built-in copy; it copies as much as the overlap between the two matrices and
-// returns the number of rows and columns it copied.
+// returns the number of rows and columns it copied. If a aliases the receiver
+// and is a transposed Dense or Vector, with a non-unitary increment, Copy will
+// panic.
```

Comment:
Initially I was going to prohibit aliased transpose copies completely, but then realised that this prevents doing row vector copies (where the row is copied from one part of a matrix to another that overlaps via a vector). This problem arises because vectors are column vectors and so thr copy must bee dont with v.T() as the source.